### ISSUE TITLE
cptStatusSheet - For Each Custom Field must have AutomaticallyRolldownToAssn = True

### ISSUE THREAD

**Issue by AronGahagan** [AronGahagan/cpt-dev#229]
## Problem:
If user selects a custom field for which `AutomaticallyRolldownToAssn = False`, then Assignments associated with the Item's Tasks will not be included in the copy/paste.

## Solution:
On `cmdCreate_click` run the following:
```vba
Application.CustomFieldPropertiesEx FieldID:=Me.cboEach.Value, AutomaticallyRolldownToAssn:=True
```

## Todo:
- [x] assign this issue
- [x] label the issue for enhancement or bug
- [x] label the issue with codemodule (so fixes can be aggregated and hotfixed together)
- [x] checkout `develop` and create `topic` branch
- [x] design, code and test
- [x] update codemodule <cpt_version>x.y.z</cpt_version>
- [x] reset BLN\_TRAP\_ERRORS
- [x] update CurrentVersions.xml **manually**
- [x] commit changes prefix comment with `issueXX`
- [x] merge `topic` into `develop` and push
- [x] if hotfix: merge `develop` into `master` and push
- [x] if feature: wait for release, do **not** merge into master
- [x] delete topic branch

**Comment by AronGahagan** [comment]
What if the selected `For Each` field is not a custom field (and therefore has no exposed way to roll down values)? Need to prompt the user, then what...roll it down manually per task? This has to happen before the filter is applied before the copy/paste. What is the best way to do this...

**Comment by AronGahagan** [comment]
Strangely, the `Contact` field rolls down even though there is no equivalent Assignment property, and the AutoFilter includes the Assignments. Perhaps it's only the Custom Fields that require the `AutomaticallyRolldownToAssn` property to be set manually? The rest of the fields in the `cboEach` ComboBox are Custom Fields so the `CustomFieldPropertiesEx` method can be implemented.
